fdbclient/BackupContainerFileSystem.actor.cpp

```diff
@@ -1349,7 +1350,9 @@ Future<Void> BackupContainerFileSystem::expireData(Version expireEndVersion,
 	    Reference<BackupContainerFileSystem>::addRef(this), expireEndVersion, force, progress, restorableBeginVersion);
 }
 
-ACTOR static Future<KeyRange> getSnapshotFileKeyRange_impl(Reference<BackupContainerFileSystem> bc, RangeFile file) {
+ACTOR static Future<KeyRange> getSnapshotFileKeyRange_impl(Reference<BackupContainerFileSystem> bc,
+                                                           RangeFile file,
+                                                           Optional<Database> cx) {
```

Comment:
Is `Database cx`  allowed to be empty? If yes, what is the usecase for it, even with no encryption enabled, `cx` can be valid?

Reply:
Yes sometimes we can run a codepath during a unit test where the `Database cx` is empty, hence the needed to make it optional